Lib/ufo2ft/__init__.py

```diff
@@ -12,7 +12,7 @@
 
 def compileOTF(ufo, outlineCompilerClass=OutlineOTFCompiler,
                featureCompilerClass=FeatureOTFCompiler, mtiFeaFiles=None,
-               kernWriter=KernFeatureWriter, markWriter=MarkFeatureWriter,
+               kernWriterClass=KernFeatureWriter, markWriterClass=MarkFeatureWriter,
```

Comment:
👍 there are a couple other renames that would make sense actually, let me commit that.